package.json

```diff
@@ -57,44 +57,47 @@
   },
   "private": false,
   "dependencies": {
-    "@angular/animations": "~7.2.0",
-    "@angular/common": "~7.2.0",
-    "@angular/compiler": "~7.2.0",
-    "@angular/core": "~7.2.0",
-    "@angular/forms": "~7.2.0",
-    "@angular/http": "~7.2.0",
-    "@angular/platform-browser": "~7.2.0",
-    "@angular/platform-browser-dynamic": "~7.2.0",
-    "@angular/platform-server": "^7.2.0",
+    "@angular/animations": "~7.2.11",
+    "@angular/common": "~7.2.11",
+    "@angular/compiler": "~7.2.11",
+    "@angular/core": "~7.2.11",
+    "@angular/forms": "~7.2.11",
+    "@angular/http": "~7.2.11",
+    "@angular/platform-browser": "~7.2.11",
+    "@angular/platform-browser-dynamic": "~7.2.11",
+    "@angular/platform-server": "^7.2.11",
     "@angular/pwa": "^0.12.0",
-    "@angular/router": "~7.2.0",
-    "@angular/service-worker": "~7.2.0",
+    "@angular/router": "~7.2.11",
+    "@angular/service-worker": "~7.2.11",
     "@compodoc/compodoc": "^1.1.8",
     "@ng-bootstrap/ng-bootstrap": "^4.0.1",
     "@ng-select/ng-select": "^2.13.2",
-    "@ngrx/effects": "~7.0.0",
-    "@ngrx/router-store": "~7.0.0",
-    "@ngrx/store": "~7.0.0",
+    "@ngrx/effects": "~7.2.0",
```

Comment:
Please upgrade all `ngrx` dependencies to `7.4.0` (see https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/1524#issuecomment-478643141)